Neos.Utility.Lock/Classes/FlockLockStrategy.php

```diff
@@ -132,7 +132,7 @@ protected function tryToAcquireLock($exclusiveLock)
      * @param boolean $exclusiveLock
      * @throws LockNotAcquiredException
      */
-    protected function applyFlock($exclusiveLock)
+    protected function applyFlock(bool $exclusiveLock)
```

Comment:
While we are at it, and since `void` as return type is only supported in PHP 7.1, why not add the missing `@return` annotation here? ;)